src/IBM.Watson.Common/Common.cs

```diff
@@ -27,7 +27,7 @@ public partial class Common
         /// <summary>
         /// The SDK version.
         /// </summary>
-        public const string Version = "watson-apis-dotnet-standard-sdk-3.5.0";
+        public const string Version = "watson-apis-dotnet-standard-sdk-4.0.0";
```

Comment:
This should stay at `3.5.0` for now. Otherwise, bumpversion won't be able to find the right version number in this file and fail, which will stop CI and the release.